metrics/parse_tfm.py

```diff
@@ -64,11 +64,14 @@ def __init__(self, start_char, end_char, char_info, width_table,
         self.ligkern_program = LigKernProgram(ligkern_table)
         self.kern_table = kern_table
 
-    def get_char_metrics(self, char_num):
+    def get_char_metrics(self, char_num, zero_indexed=False):
         if char_num < self.start_char or char_num > self.end_char:
             raise RuntimeError("Invalid character number")
 
-        info = self.char_info[char_num + self.start_char]
+        if zero_indexed:
+            info = self.char_info[char_num - self.start_char]
+        else:
+            info = self.char_info[char_num + self.start_char]
```

Comment:
The script font, rsfs, is zero_indexed meaning that although the start_char is 65, the first glyph index is 0.  TODO: update the mapping so that we don't have to do ```- self.start_char``` here.